.travis.yml

```diff
@@ -10,6 +10,10 @@ env:
   - VERSION=0.12 PYTHON_VERSION=3.6 VARIANT=slim
   - VERSION=0.12 PYTHON_VERSION=2.7 VARIANT=alpine
   - VERSION=0.12 PYTHON_VERSION=3.6 VARIANT=alpine
+  - VERSION=1.0.3 PYTHON_VERSION=2.7 VARIANT=slim
```

Comment:
I would drop the patch version from the build matrix. This is inline with how we're handling the `0.x` series in this repo, and will allow applications utilizing the `1.0` series image to automatically receive patch updates, etc.

Reply:
Oh cool! I will do that.